invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/BookDetails/components/BookTab/BookTab.js

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from 'react';
+import {
+  Icon,
+  Tab,
+  Accordion,
+  Segment,
+  Menu,
+  Grid,
+  Responsive,
+} from 'semantic-ui-react';
+
+class BookAccordion extends Component {
```

Comment:
I would put the `BookAccordion` component in its own file. One component per file with class name matching the file name.